packages/playground/src/App.js

```diff
@@ -147,6 +148,10 @@ export default class extends React.Component {
     this.setState(() => ({ botAvatarInitials: value }));
   }
 
+  handleDisconnectClick() {
+    this.props.store.dispatch({ type: 'DIRECT_LINE/DISCONNECT' });
```

Comment:
Can you reference a constant here instead of `'DIRECT_LINE/DISCONNECT'`?

Reply:
Currently, we did not expose any action type (constant).

We need a better plan on how/what to expose. Exposing everything will hurt the community than not exposing them at all.

Resolving this for now. If you think expose is important, please file a new bug and we discuss over there.